ansible/inference/deploy-worker.yaml

```diff
@@ -0,0 +1,40 @@
+# Ansible playbook to setup inference worker
+
+- name: Deploy inference
+  hosts: dev
+  gather_facts: true
+  vars:
```

Comment:
in here we also need the "parallelism" variable defined in the worker-full dockerfile